### PR TITLE
Make json keys and DB columns consistent

### PR DESCRIPTION
**Pull request by yhabteab**

### CODE REVIEW

File: schema/mysql/upgrades/1.0.0.sql [hunk 1]
```diff
@@ -164,10 +164,14 @@ ALTER TABLE notification_history
...
-  ADD COLUMN previous_soft_state tinyint unsigned NOT NULL AFTER hard_state;
+  ADD COLUMN previous_soft_state tinyint unsigned NOT NULL AFTER hard_state,
+  CHANGE attempt check_attempt tinyint unsigned NOT NULL,
+  CHANGE timeout check_timeout int unsigned DEFAULT NULL;
 
 ALTER TABLE service_state
-  ADD COLUMN previous_soft_state tinyint unsigned NOT NULL AFTER hard_state;
+  ADD COLUMN previous_soft_state tinyint unsigned NOT NULL AFTER hard_state,
+  CHANGE attempt check_attempt tinyint unsigned NOT NULL,
+  CHANGE timeout check_timeout int unsigned DEFAULT NULL;
```
Comment (Al2Klimov) [inline schema/mysql/upgrades/1.0.0.sql:174]: I guess RENAME isn’t old enough?

**Review by Al2Klimov — APPROVED** [review]

File: schema/mysql/upgrades/1.0.0.sql (hunk 1 above)
Comment (yhabteab) [reply schema/mysql/upgrades/1.0.0.sql:174]: Yes, it' s actually only available since mysql 8.

**Comment by julianbrost** [comment]
@yhabteab Is there another reason why this is marked as draft apart from the PR in icinga2 not being merged yet?

**Comment by Al2Klimov** [comment]
Wrong collocutor :P But no, no other reason.

**Pull request merged**